src/main/java/io/mosip/digitalcard/service/impl/PDFCardServiceImpl.java

```diff
@@ -0,0 +1,568 @@
+	 */
+	private Map<String, byte[]> getDocuments(String credential, String credentialType, String encryptionPin,
+			String requestId,
+			String cardType,
+			boolean isPasswordProtected) {
+		printLogger.debug("PrintServiceImpl::getDocuments()::entry");
+		String credentialSubject;
+		Map<String, byte[]> byteMap = new HashMap<>();
+		String uin = null;
+		String rid=null;
+		String password = null;
+		boolean isPhotoSet=false;
+		String individualBio = null;
+		Map<String, Object> attributes = new LinkedHashMap<>();
+		boolean isTransactionSuccessful = false;
+		String template = uinCardTemplate;
+		byte[] pdfbytes = null;
+		try {
+			credentialSubject = getCrdentialSubject(credential);
+			org.json.JSONObject decryptedJson =new org.json.JSONObject(credentialSubject);
+			if(decryptedJson.has("biometrics")){
+				individualBio = decryptedJson.getString("biometrics");
+				String individualBiometric = new String(individualBio);
+				isPhotoSet = setApplicantPhoto(individualBiometric, attributes);
+				attributes.put("isPhotoSet",isPhotoSet);
+			}
+			uin = decryptedJson.getString("UIN");
+			if (isPasswordProtected) {
+				password = getPassword(uin);
+			}
+			if (credentialType.equalsIgnoreCase("qrcode")) {
+				boolean isQRcodeSet = setQrCode(decryptedJson.toString(), attributes,isPhotoSet);
+				InputStream uinArtifact = templateGenerator.getTemplate(template, attributes, templateLang);
+				pdfbytes = uinCardGenerator.generateUinCard(uinArtifact, UinCardType.PDF,
+						password);
+
+			} else {
+			if (!isPhotoSet) {
+				printLogger.debug(DigitalCardServiceErrorCodes.APPLICANT_PHOTO_NOT_SET.name());
+			}
+			setTemplateAttributes(decryptedJson.toString(), attributes);
+			attributes.put(IdType.UIN.toString(), uin);
+			byte[] textFileByte = createTextFile(decryptedJson.toString());
+			byteMap.put(UIN_TEXT_FILE, textFileByte);
+			boolean isQRcodeSet = setQrCode(decryptedJson.toString(), attributes,isPhotoSet);
+			rid=getRid(decryptedJson.get("id"));
+			if (!isQRcodeSet) {
+				printLogger.debug(DigitalCardServiceErrorCodes.QRCODE_NOT_SET.name());
+			}
+			// getting template and placing original valuespng
+			InputStream uinArtifact = templateGenerator.getTemplate(template, attributes, templateLang);
+			if (uinArtifact == null) {
+				printLogger.error(DigitalCardServiceErrorCodes.TEM_PROCESSING_FAILURE.name());
+				throw new DigitalCardServiceException(
+						DigitalCardServiceErrorCodes.TEM_PROCESSING_FAILURE.getErrorCode(),DigitalCardServiceErrorCodes.TEM_PROCESSING_FAILURE.getErrorMessage());
+			}
+
+			pdfbytes = uinCardGenerator.generateUinCard(uinArtifact, UinCardType.PDF, password);
+				InputStreamResource resource = new InputStreamResource(new ByteArrayInputStream(pdfbytes));
+				  File pdfFile = new File("src/main/resources/uin.pdf");
+				  OutputStream os = new FileOutputStream(pdfFile);
+				  os.write(pdfbytes);
+				  os.close();
+		}
+			printStatusUpdate(requestId, pdfbytes, credentialType,rid);
```

Comment:
This service has no direct relevant to print, lets remove print reference everywhere. Also the status update should be done by the digitial card service and not pdf card service

Reply:
taken care

Reply:
There are few properties still named as print